src/Views/OutputPrint.js

```diff
@@ -0,0 +1,48 @@
+import { Console } from '@woowacourse/mission-utils';
+import { PrizeDescription } from '../Constants/prizeConfig.js';
+import { OutputComment } from '../Constants/display.js';
+import getNumberWithComma from '../Utils/getNumberWithComma.js';
+
+const OutputPrint = {
+  message: (message) => {
+    Console.print(message);
+  },
+
+  error: (errorMessage) => {
+    Console.print(errorMessage);
+  },
```

Comment:
```suggestion
  message: (message) => {
    Console.print(message);
  },

  error: (errorMessage) => {
  // 물론 이미 [ERROR] 접두사가 있지만 위에 print 메서드랑 무언가 달라야 한다는 것을 예시로 보여주기 위해 추가하였습니다:) 
    Console.print(`[ERROR] ${errorMessage}`);
  },
```
두 코드의 차이점은 뭐라고 생각하시나요?? 저는 개인적으로 이렇게 둘이 나눌거면 다른 무언가가 있어야 한다는 생각이 드는 것 같네요🤔

Reply:
저는 함수의 이름으로 에러 메세지를 출력한다는 걸 읽는 사람에게 전달하고 싶어서 일부러 분리했습니다!🫠 `OutputPrint.message()`로 정상 메세지를 출력하고, `OutputPrint.error()`로 여기 매세지는 뭔가 잘못됐다는 걸 제시하고 싶었거든요.

지금 살펴보면 `OutputPrint.message(error.message)`였을테니 굳이 분리할 필요는 없었겠네요🥲